robo.yml

```
@@ -9,49 +9,49 @@ backend:
 data:
   - name: "corporate-body-classification"
     graph: "http://publications.europa.eu/resource/authority/corporate-body-classification"
-    url: "http://publications.europa.eu/resource/cellar/b8924e2d-ce0b-11eb-ac72-01aa75ed71a1.0001.04/DOC_1"
+    url: "http://publications.europa.eu/resource/cellar/947c85d5-581c-11ec-91ac-01aa75ed71a1.0001.01/DOC_1"
     format: "rdf"
 
   - name: "corporate-body"
     graph: "http://publications.europa.eu/resource/authority/corporate-body"
-    url: "http://publications.europa.eu/resource/cellar/1e09ca83-594b-11ec-91ac-01aa75ed71a1.0001.03/DOC_1"
+    url: "http://publications.europa.eu/resource/cellar/dca0f07d-787e-11ec-9136-01aa75ed71a1.0001.03/DOC_1"
     format: "rdf"
 
   - name: "country"
     graph: "http://publications.europa.eu/resource/authority/country"
-    url: "http://publications.europa.eu/resource/cellar/acb33658-ce0b-11eb-ac72-01aa75ed71a1.0001.04/DOC_1"
+    url: "http://publications.europa.eu/resource/cellar/820161ea-581c-11ec-91ac-01aa75ed71a1.0001.04/DOC_1"
     format: "rdf"
 
   - name: "digital-europa-thesaurus"
     graph: "http://data.europa.eu/uxp"
-    url: "http://publications.europa.eu/resource/cellar/b451491c-ce0b-11eb-ac72-01aa75ed71a1.0001.03/DOC_1"
+    url: "http://publications.europa.eu/resource/cellar/76166330-581c-11ec-91ac-01aa75ed71a1.0001.01/DOC_1"
     file: "DET-skos-core.rdf"
     format: "zip"
 
   - name: "eurovoc-thesaurus"
     graph: "http://publications.europa.eu/resource/dataset/eurovoc"
-    url: "http://publications.europa.eu/resource/cellar/b868cf85-c47b-11eb-a925-01aa75ed71a1.0001.04/DOC_1"
+    url: "http://publications.europa.eu/resource/cellar/bcd714b1-5f05-11ec-9c6c-01aa75ed71a1.0001.03/DOC_1"
```

Here also should be used correct file (as before)
```suggestion
    url: "http://publications.europa.eu/resource/cellar/bcd714b1-5f05-11ec-9c6c-01aa75ed71a1.0001.04/DOC_1"
```
Currently, image event can be build with http://publications.europa.eu/resource/cellar/bcd714b1-5f05-11ec-9c6c-01aa75ed71a1.0001.03/DOC_1